server/db/model/UserBadge.js

```diff
@@ -0,0 +1,21 @@
+module.exports = (sequelize, type) => sequelize.define('userbadge', {
```

Comment:
is this this correct case?

Reply:
I am not sure I wanted to ask you or Elliot, so I just went with the simplest guess I can change to Camel case or is it snake case?